requirements/base.in

```diff
@@ -1,5 +1,5 @@
-open-api-framework[celery,cors,markup,geo,csp,commonground,inclusions,sanitization,server,redis]
-django-csp==3.8.0
+git+https://github.com/maykinmedia/open-api-framework.git@feature/178-support-structlog#egg=open-api-framework[celery,cors,markup,geo,csp,commonground,inclusions,sanitization,server,redis,structlog-celery]
+# django-csp==3.8.0
```

Comment:
This can be removed, because the next release of OAF will fix compatibility with 4.0
```suggestion
```